README.md

```diff
@@ -20,14 +20,34 @@ npm install react-transition-group --save
 yarn add react-transition-group
 ```
 
+## CDN / External
+
+To exclude the react-transition-group package from your primary build and use an external CDN when bundling, link
+to the following CDN:
+
+[https://unpkg.com/react-transition-group/dist/react-transition-group.min.js](https://unpkg.com/react-transition-group/dist/react-transition-group.min.js)
+
+In order to exclude react-transition-group using **Webpack**, include the following configuration in the `externals`
+webpack configuration section: 
+
+```
+externals: {
+    'react-transition-group': {
+        amd: 'react-transition-group',
+        commonjs2: 'react-transition-group',
+        commonjs: 'react-transition-group',
+        root: 'ReactTransitionGroup',
+    }
+```
+
 ## High-level API: CSSTransitionGroup
 
 `CSSTransitionGroup` is a high-level API based on [`TransitionGroup`](#low-level-api-transitiongroup) and is an easy way to perform CSS transitions and animations when a React component enters or leaves the DOM. It's inspired by the excellent [ng-animate](http://www.nganimate.org/) library.
 
 **Importing**
 
 ```javascript
-import CSSTransitionGroup from 'react-transition-group/CSSTransitionGroup' // ES6
+import { CSSTransitionGroup } from 'react-transition-group' // ES6
```

Comment:
I wasn't able to get the cherry picked version to exclude properly when testing.  All documentation everywhere prefers the standard ES6 brace style for non-default components. Do you have an example webpack external config where using cherry picked version will work?